modules/ROOT/pages/test-waf-policy.adoc

```diff
@@ -0,0 +1,299 @@
+= Test the WAF Policy
+
+If the following request is sent in when the WAF policy shown in the “Request RuleSets” has been deployed, the bad JSON in this request will generate a WAF error. The closing double quote for the textField value has been omitted.
+
+----
+curl -v -k -X POST  --resolve choice-waf14.example.com:443:192.168.2.1  https://choice-waf14.example.com:443/ -H "Content-Type: text/plain" -H "SOAPAction: 'http://example.org/soapformat/Example'" -H "User-Agent: (hydra)" -H "Content-Length: 10" -d 'hello'
+----
+
+The response would be:
```

Comment:
This does not appear to be a json request since content-type is set to text/plain. And, we don't want to have any examples that rely on the xml or json parser since these parsers are always disabled for now.